client/blocks/product-selector/docs/example.jsx

```diff
@@ -15,15 +15,41 @@ const products = [
 		title: 'Jetpack Backup',
 		description: (
 			<p>
-				Automatic scanning and one-click fixes keep your site one step ahead of security threats.{' '}
-				<a href="https://jetpack.com/">More info</a>
+				Always-on backups ensure you never lose your site. Choose from real-time or daily backups.{' '}
+				<a href="https://jetpack.com/">Which one do I need?</a>
 			</p>
 		),
 		id: 'jetpack_backup',
 		options: {
 			yearly: [ 'jetpack_backup_daily', 'jetpack_backup_realtime' ],
 			monthly: [ 'jetpack_backup_daily_monthly', 'jetpack_backup_realtime_monthly' ],
 		},
+		optionDescriptions: {
```

Comment:
We could totally omit this from the example. There are other options that the example is missing and maybe it makes sense to keep it simple.